.yamato/upm-ci.yml

```diff
@@ -186,24 +185,8 @@ testStandalone_{{ platform.name }}_{{ editor.version }}:
 
 test_trigger:
   name: TestsTrigger
```

Comment:
Does that allow this job to run on any available platform?

Reply:
Ahh yes. by removing the agent, this job got tagged a a "virtual" job (basically a dependency gathering handle). There are no VMs associated with virtual jobs.